src/main/resources/sql_scripts/2second.sql

```diff
@@ -6,10 +6,12 @@ INSERT INTO address ( street, home_num, district_id, description) VALUES
                                                                       ( 'ul. Słoneczna', '5', 1, 'Biuro PJATK'),
                                                                       ( 'ul. Parkowa', '7', 3, 'Wejście przez zieloną bramę, na domofonie #31');
 
-INSERT INTO "user" ( firstname, lastname, email, phone_number, role_id, is_pesel_verified, is_agreement_signed, is_adult,password_hash,salt) VALUES

```

Comment:
to powinno być w osobnym pliku. Nie edytujemy plików liquibase'a, które już trafiły do maina.